CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+# **CODE OF CONDUCT**
+
+
+**All participants of Frontendweekly are expected to abide by our Code of Conduct, both online and during in-person events that are hosted and/or associated with Frontendweekly.**
+
+## **The Pledge**
+
+In the interest of fostering an open and welcoming environment, we pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+
+## **The Standards**
+
+**Examples of behaviour that contributes to creating a positive environment include:**
```

Comment:
This is great. I think this can be expounded further. Draw some inspiration from the [Django Foundation's Code of Conduct](https://www.djangoproject.com/conduct/) as well as this line from our README `In the meantime, be kind, compassionate, patient and respectful to everyone here`

Reply:
Okay. On it.